.githooks/README.md

```diff
@@ -0,0 +1,28 @@
+# .githooks
+
+This folder contains executable files that will be invoked by git at certain git operations.
+
+By default git hooks are located i `.git/hooks` folder at the root of the repository. Since the default folder is hidden by most IDEs, this repository reconfigures git's hook location in order to make the hooks visible and easier to maintain.
```

Comment:
Some people have default git templates which populate git hooks for them (including me), do we know of any way to preserve this?

For me, it doesn't cause many problems, since it just generates tags files that I don't use (LSP uses other mechanism for goto-ref), but others may have some custom git hooks necessary for their work environment

Reply:
Does your git template populate the default `.git/hooks` folder? Which git hooks do you have implemented?

Reply:
```bash
$ ls .git/hooks/
ctags  post-checkout  post-commit  post-merge  post-rewrite
```

Reply:
Example with me adding `echo "hello world"` to my personal `post-commit` githook
```
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ git commit -m "test"
hello world
[AAP-35592 d4af689] test
 1 file changed, 1 insertion(+)
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ make 
build             check             check_black       check_flake8      check_isort       clean             git_hooks_config  help              help/             lint              postgres          PYTHON_VERSION    stop-postgres
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ make git_hooks_config 
git config --local core.hooksPath .githooks
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ git add .
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ git commit -m test

Running user pre commit for 
On branch AAP-35592
nothing to commit, working tree clean
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ git status
On branch AAP-35592
nothing to commit, working tree clean
bpaciore@bpaciore-thinkpadp16vgen1:~/src/django-ansible-base$ 
```

Reply:
I'm going to resolve this for now as per my recent comment, then approve, so we have parity.